data/libwacom.stylus

```diff
@@ -291,6 +291,22 @@ Buttons=2
 Axes=Tilt;Pressure;Distance;
 Type=General
 
+[0x200]
+# Cintiq Pro 2022
+Name=Pro Pen 3
+Group=cintiqpro2022
+Buttons=3
+Axes=Tilt;Pressure;Distance;
+Type=General
+
+[0x04200]
```

Comment:
@Joshua-Dickens We added an extra 0 at the 4th nibble position for the actual ID in the kernel driver. So, the correct ID here should be 0x40200.